packages/toolpad-app/src/toolpad/AppEditor/PageEditor/UrlQueryEditor.tsx

```diff
@@ -83,10 +106,14 @@ export default function UrlQueryEditor({ pageNodeId }: UrlQueryEditorProps) {
         <DialogTitle>Edit page parameters</DialogTitle>
         <DialogContent>
           <Typography>
-            The parameters you define below will be made available in bindings under the{' '}
-            <code>page.parameters</code> global variable. You can set these parameters in the url
-            with query variables (<code>?param=value</code>).
+            Page parameters allow you to pass external data into the Toolpad page state via the URL
+            query. Read more in the{' '}
+            <Link href="https://mui.com/toolpad/concepts/managing-state/#page-parameters">
```

Comment:
We are missing `target="_blank"`. I lost the state of my app when clicking on the link 🙃 Fixed in 1e00b651120eff90f568b6d070270a808927a805.

Reply:
This just happened to me! Thanks for fixing it.